root/static/scripts/guess-case/utils.js

```diff
@@ -87,6 +87,14 @@ var lowerCaseBracketWordsList = [
 
 var lowerCaseBracketWords = new RegExp('^(' + lowerCaseBracketWordsList.join('|') + ')$', 'i');
 
+exports.turkishUpperCase = function(str) {
+  return str.replace(/i/g, 'İ').toUpperCase();
+};
+
+exports.turkishLowerCase = function(str) {
+  return str.replace(/I\u0307/g, 'i').replace(/I/g, 'ı').replace(/İ/g, 'i').toLowerCase();
```

Comment:
OK, it seems the test runner (i.e. node.js) indeed behaves that strangely. Leave the third `replace` in, then. :-)